docs/DEVELOPMENT.md

```diff
@@ -94,30 +96,30 @@ advanced via a fast-forward merge after successful testing by the CI job
 that rebuilds and tests the `devel` branch.
 
 Code changes are made via pull requests against `devel`. Each of them
-will get tested separately by the CI system before merging, but only a
+will be tested separately by the CI system before merging, but only a
 subset of the tests can be run due to time constraints.
 
 Beware that after merging one PR, the existing pre-merge tests results
 for other PRs become stale because they were based on the old `devel`
 branch. Because `devel` is allowed to be less stable than `master`, it
 is okay to merge two PRs quickly after one another without
-retesting. If two PRs that merged that don't have code conflicts
-(which would get detected by GitHub) but which nonetheless don't work
+retesting. If two merged PRs don't have code conflicts
+(which would be detected by GitHub\*) but nonetheless don't work
 together, the combined testing in the `devel` branch will find
-that. This will block updating `master` and thus needs to be dealt
+that. This block updating `master` and thus needs to be dealt with
 quickly.
 
 Releases are created by branching `release-x.y` from `master` or some
 older, stable revision. The actual `vx.y.z` release tags are set
 on revisions in the corresponding `release-x.y` branch.
 
 Releases and the corresponding images are never changed. If something
-goes wrong after setting a tag (like detecting a bug while testing the
+goes wrong after setting a tag (such as detecting a bug while testing the
 release images), a new release is created.
 
 Container images reference a fixed base image. To ensure that the base
-image remains secure, it gets scanned for known vulnerabilities regularly
-and a new release is prepared manually if needed. The new release then
+image remains secure, it is scanned for known vulnerabilities regularly
+and a new release is prepared manually, if needed. The new release then
```

Comment:
This whole paragraph is factual incorrect. We replaced this approach with fixed base images a while back and just forgot to update the text. Can you replace the entire paragraph with:
```
Container images reference a floating base image. Whatever version of that was current at the time of building the PMEM-CSI image then serves as base for the release. To ensure that the release image remains secure, it is scanned for known vulnerabilities regularly
and a new release is prepared manually, if needed. The new release then uses a newer base image.
```